index.html

```diff
@@ -47,97 +48,40 @@ <h2 class="connect_wrapper"><a class="contact" href="#">LETS CONNECT</a></h2>
     </section>
     
     <section class="grid__wrapper" id="portfolio">
-        <div class="covers">
-            <img class="img__class" id="pic__2" src="./img/pic1.png" alt="">
-            <img class="img__class" id="pic__1" src="./img/pic1.png" alt="">
-            <div class="right__box">
-                <h1 class="bold-text">Tonic</h1>
-                <ul class="selections">
-                    <li class="main__txt__wrapper"><a href="#" class="main__txt"></a>CANOPY</li>
-                    <li class="circle__wrapper"><a href="#" class="circle"></a></li>
-                    <li class="list__wrapper"><a href="#" class="list">Back End Dev</a></li>
-                    <li class="circle__wrapper"><a href="#" class="circle"></a></li>
-                    <li class="list__wrapper"><a href="#" class="list">2015</a></li>
-                </ul>
-                <p class="signup-disc">A daily selection of privately personalized reads: no acounts or signup's required</p>
-                <ul class="platform">
-                    <li class="pla__btn">html</li>
-                    <li class="pla__btn">css</li>
-                    <li class="pla__btn">javascript</li>
-                </ul>
-                <button class="see-project">See Project</button>
```

Comment:
Please create a pop-up for the desktop while following the design. The image looks blurry and scattered. Pop-up should not have full width. Kindly see the screenshot below for reference
![desktop-popup](https://user-images.githubusercontent.com/35504149/136222730-da3b8a0a-94c5-42c1-985e-6c24effb00c0.png)
.